README.md

```diff
@@ -42,15 +34,17 @@
 
 ## Getting the app running
 
-- Once you have completed the environment setup and have the project cloned, we're ready to get the app running.
+- Once you have completed the environment setup and have the project cloned, you're ready to get the app running.
 - Navigate to the root folder of the project in a terminal and perform a `yarn install`. We're using yarn in this project instead of npm, so you should not be using npm to execute any commands.
   - If you do not have yarn installed, install it either via npm or brew (instructions [here](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable))
-- If you are running on an M1/M2 macbook, run this command before dealing with ruby/bundler/pods: `env /usr/bin/arch -arm64 /bin/zsh --login`. If any of the steps with ruby or bundler fail, try prefixing them with arch -arm64 (e.g. arch -arm64 bundle install)
-- Install [RVM](https://rvm.io/) - (if you haven't already - it allows you to have different Ruby versions for different projects/directories on your machine).
+- If you are running on an M1/M2 macbook, you may need to run this command before dealing with ruby/bundler/pods: `env /usr/bin/arch -arm64 /bin/zsh --login`. If any of the steps with ruby or bundler fail, try prefixing them with `arch -arm64` (e.g. `arch -arm64 bundle install`)
+- Install [RVM](https://rvm.io/) if you haven't already. This allows you to have different Ruby versions for different projects/directories on your machine.
 - Navigate to the ios folder (`cd ios`) and perform `bundle install`
   - If you get the error _'your ruby version is x but your gemfile is y'_, try entering these commands: `rvm install y` followed by `rvm use y`, then retry the `bundle install`.
 - Then perform a `bundle exec pod install`
   - If you get an error about compatible versions of firebase/coreOnly, run `bundle exec pod install --repo-update`
+- Drag and drop the NetSkope cert onto both your Android emulator and iOS simulator. On your Mac, this can be found at `Library/Application Support/Netskope/STAgent/data/nscacert.pem`
```

Comment:
Fixed path 
```suggestion
- Drag and drop the NetSkope cert onto both your Android emulator and iOS simulator. On your Mac, this can be found at `/Library/Application\ Support/Netskope/STAgent/data/nscacert.pem`
```